src/open_inwoner/utils/ckeditor.py

```diff
@@ -105,3 +118,18 @@ def get_product_rendered_content(product):
                 element.append(icon)
 
     return soup
+
+
+def convert_styling_to_nlds(response: requests.Response) -> None:
+    """
+    Post-process rendered HTML content, replace legacy styling with NLDS classes
+    """
+    raw_html = response.content
+    html_tree = html.fromstring(raw_html)
+
+    for tag, style_class in NLDS_MAP.items():
+        elems = html_tree.cssselect(tag)
```

Comment:
Let's use `.iterfind(tag)` to find node names: it is an iterator and doesn't involve a CSS engine to find a tag name.